### PR TITLE
Adds a general troubleshooting page

### PR DESCRIPTION
**Pull request by prees1**
Adds page and includes suggestion on receiving emails with outlook.

### CODE REVIEW

File: pages/tidal tools/general-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+keywords: troubleshooting, faq
+last_updated: Apr 13, 2020
+summary: If you are having problems check here and see if these solutions can help.
+sidebar: main_sidebar
+permalink: general-troubleshooting.html
+---
+
+## Why aren't I receiving verification or invite emails?
+
+It is possible the emails are being blocked by your spam or filtering system. If you are using Outlook for email we suggest you check both your spam folder and the ['quarantine'](https://protection.office.com/quarantine).
```
Comment (dcolebatch) [inline pages/tidal tools/general-troubleshooting.md:12]: I suggest:
> _If you are using *Microsoft Office 365* for your email service, either online at https://outlook.office365.com/mail/ or an email client, we suggest you...._

**Comment by prees1** [comment]
@dcolebatch good call, I made that change. Any other thoughts before release?

File: pages/tidal tools/general-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+keywords: troubleshooting, faq
+last_updated: Apr 13, 2020
+summary: If you are having problems check here and see if these solutions can help.
+sidebar: main_sidebar
+permalink: general-troubleshooting.html
+---
+
+## Why aren't I receiving verification or invite emails?
+
+It is possible the emails are being blocked by your spam or filtering system. If you are using Microsoft Office 365 for your email service, either online at https://outlook.office365.com/mail/ or an email client, we suggest you check both your spam folder and the ['quarantine'](https://protection.office.com/quarantine).
```
Comment (justinbarclay) [inline pages/tidal tools/general-troubleshooting.md:12]:
```suggestion
It is possible the emails are being blocked by your spam or filtering system. If you are using Microsoft Office 365 for your email service, either online at https://outlook.office365.com/mail/ or in an email client, we suggest you check both your spam folder and the ['quarantine'](https://protection.office.com/quarantine).
```

**Comment by dcolebatch** [comment]
LGTM!

TY!

**Pull request merged**